test/20_supervisor_notification.ts

```diff
@@ -146,6 +147,7 @@ export default () => {
 
 				// Just to confirm that the MAX_SAFE_SQL_BINDS limitation is still valid
 				it('should fail when providing more parameters to $in than PG support', async function () {
+					nock.restore();
```

Comment:
I'm not sure why disabling interceptors make any difference here. I'm wondering if it is more about extra processing time due to interceptors and not necessarily the interceptors themselves causing problems?